dapp/src/utils/contracts.js

```diff
@@ -126,7 +126,7 @@ export async function setupContracts(account, library, chainId) {
   // execute in parallel and repeat in an interval
   window.fetchInterval = setInterval(() => {
     callWithDelay()
-  }, 5000)
+  }, 30000)
```

Comment:
do we really need to refresh the APR every 30sec? The APR is not going to change a lot that quickly. I would think a 1hr refresh is good enough. But maybe I'm missing something CC @micahalcorn